.go-env.sh

```diff
@@ -4,12 +4,25 @@ set -Eeuo pipefail
 dir="$(dirname "$BASH_SOURCE")"
 dir="$(readlink -ve "$dir")"
 
-user="$(id -u):$(id -g)"
+src="$dir"
+dst="$dir"
+msys=
+if [ "$(uname -o)" = 'Msys' ];	then
```

Comment:
```suggestion
if [ "$(uname -o)" = 'Msys' ]; then
```

(normally, I'd also suggest we pull out `$(uname -o)` into a variable so errors are captured appropriately, but in this case I think it's fine as-is because errors in that invocation would mean we're probably not `Msys`)